packages/paste-core/components/button/src/styles.ts

```diff
@@ -115,20 +113,26 @@ const variantPrimaryEnabled = (props: ButtonWrapperProps): SerializedStyles =>
     baseEnabled(props),
     variantPrimaryBase(props),
     css`
-      border-color: ${themeGet('borderColors.colorBorderPrimary')(props)};
       background-color: ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
+      box-shadow: inset 0 0 0 ${themeGet('space.space10')(props)}
+        ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
 
       &:hover {
-        border-color: ${themeGet('borderColors.colorBorderPrimaryDarker')(props)};
         background-color: ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)};
+        box-shadow: inset 0 0 0 ${themeGet('space.space10')(props)}
+          ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)};
       }
       &:focus {
-        border-color: ${themeGet('borderColors.colorBorderPrimaryDarker')(props)};
         background-color: ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
+        box-shadow: inset 0 0 0 ${themeGet('space.space10')(props)}
+            ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)},
+          ${themeGet('shadows.shadowFocus')(props)};
```

Comment:
`shadows.shadowFocus` does the whole rule: 
![image](https://user-images.githubusercontent.com/1592327/75569697-3c451a80-5a1b-11ea-9585-9776bd33e664.png)
this might be why we're seeing two box shadows in the snapshot maybe

Reply:
Misread the `,` - lgtm